flaskbb/forum/views.py

```diff
@@ -696,15 +841,35 @@ def post(self, post_id):
 
 
 class RawPost(MethodView):
-    decorators = [login_required]
+    decorators = [
```

Comment:
Need to ensure that a user is allowed to access the forum before resolving this.